src/Components/Popover/Popover.tsx

```diff
@@ -89,6 +99,23 @@ export class Popover extends Component<PopoverProps, State> {
   }
 
   public componentDidUpdate(prevProps: PopoverProps) {
+    if (!this.triggerReceivedHandlers) {
+      this.triggerReceivedHandlers = false
+
+      this.props.triggerRef.current.addEventListener(
```

Comment:
I think it could be fine-tuned a bit. I have this extra step in `componentDidUpdate` because when the component mounts the trigger ref doesn't exist yet and the handlers don't get added. Might make sense to just add the handlers on the first `componentDidUpdate` rather than `componentDidMount`